app/controllers/admins/kudos_controller.rb

```diff
@@ -0,0 +1,31 @@
+module Admins
+  class KudosController < ApplicationController
+    before_action :set_kudo, only: %i[destroy]
+    layout 'admin'
```

Comment:
Missing authentication of admin, so anyone can reach this pages and delete kudos. I would suggest to create AdminController that will have `layout 'admin'` and `before_action :authenticate_admins`, and inherit these two from it